Dockerfile

```diff
@@ -10,8 +10,11 @@ RUN make all
 FROM alpine:latest
 
 RUN apk add --no-cache ca-certificates
-COPY --from=builder /go/thor/bin/thor /usr/local/bin/
-COPY --from=builder /go/thor/bin/disco /usr/local/bin/
+RUN adduser -D -s /bin/ash thor
+USER thor
+ENV PATH="$PATH:/home/thor"
+COPY --from=builder /go/thor/bin/thor /home/thor
```

Comment:
Any particular concern about putting thor under the new user home directory other than `/usr/local/bin`?

Reply:
In a multi-user system, if a malicious user gains write access to `/usr/local/bin`, they could potentially replace system-wide executables with their own, leading to security issues. 

However, in a containerized environment, this concern is less relevant, as each container is isolated from others and runs a single application. Since this is not a multi-user environment, I will revert back to `/usr/local/bin` as suggested, thanks!